presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix scheduling issue in join spill to improve memory used by unspilled data.
+* Fix spilled query exceeding local memory limit due to incorrect memory size calculation when spilled data is read by SingleStreamSpiller.
+* Fix deadlock problem that results into join + limit getting stuck when spilling is enabled.
```

Comment:
Fix deadlock for queries with join and limit when spilling is enabled.

Reply:
done